test/e2e/certificates_test.go

```diff
@@ -592,11 +592,11 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "clusterissuer.yaml"), ns.Name)
 			defer loader.DeleteFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "clusterissuer.yaml"), ns.Name)
 
-			By("creating an openshift-hello deployment")
+			By("creating an hello-openshift deployment")
```

Comment:
This is good, but let's add all the non-`PollUntilContextTimeout` changes in a separate commit please.